app/components/attribute_help_texts/show_dialog_component.html.erb

```diff
@@ -0,0 +1,33 @@
+<%=
+  render(Primer::Alpha::Dialog.new(id: dialog_id, title:, size: :large)) do |dialog|
+    dialog.with_header(variant: :large)
+
+    dialog.with_body do
+      concat(
+        render(Primer::Box.new(mt: 1, classes: "op-uc-container op-uc-container__no-permalinks")) do
+          helpers.format_text(@attribute_help_text, :help_text)
+        end
+      )
+
+      if has_attachments?
+        concat render(Primer::Beta::Heading.new(tag: :h4)) { I18n.t(:label_attachment_plural) }
+        concat helpers.list_attachments(resource_representer, inputs: { allowUploading: false })
+      end
+    end
+
+    dialog.with_footer(show_divider: true) do
```

Comment:
I am not a fan of the divider but I see the benefit it brings for longer help texts. As discussed, I am fine with keeping it for now and review once the UX team is back.